devimint/src/main.rs

```diff
@@ -553,13 +553,14 @@ async fn cli_tests(dev_fed: DevFed) -> Result<()> {
     info!("Testing Client-NG");
     // ## reissue e-cash
     info!("Testing reissuing e-cash");
+    const CLIENT_NG_INITIAL_BALANCE: u64 = 10_000_000; // from initial pegin
```

Comment:
tests shouldn't have to deal with auto pegin.

should do pegin in dev-fed subcommand handler, not in `dev_fed` function

Reply:
OK that makes sense thanks.

I made a naive attempt as follows but then it breaks the old client pegin when running `just mprocs`:
```diff
@@ -1176,8 +1165,9 @@ async fn main() -> Result<()> {
         }
         Cmd::DevFed => {
             let (process_mgr, task_group) = setup(args.common).await?;
-            let _daemons =
+            let daemons =
                 write_ready_file(&process_mgr.globals, dev_fed(&process_mgr).await).await?;
+            daemons.fed.pegin_ng(10_000).await?;
             task_group.make_handle().make_shutdown_rx().await.await?;
         }

```
I'll be travelling a bit for the next few days so will try to resolve soon.

Reply:
do pegin before calling write_ready_file

Reply:
Ah yes thanks, that fixed it :)